ansible/roles/jitsi-meet/templates/config.js.j2

```diff
@@ -24,14 +24,17 @@ var config = {
     },
 
 {% if jitsi_meet_token_auth_url %}
-    tokenAuthUrl: '{{jitsi_meet_token_auth_url}}#room={room}&subdir=' + subdomain,
+    tokenAuthUrl: {{jitsi_meet_token_auth_url}},
```

Comment:
Need quotes?

Reply:
The quotes are in the tokenAuthUrl property because to be able to set stuff like:

```
'{{jitsi_meet_token_auth_url}}#room={room}&subdir=' + subdomain,
```